notebooks/233-encodec-audio-compression/233-encodec-audio-compression.ipynb

```diff
@@ -123,6 +167,37 @@
     "ipd.Audio(sample_file)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "970e162d-ea7e-4d32-bb35-3c3ed29f9720",
+   "metadata": {},
+   "source": [
+    "### Preprocessing\n",
+    "\n",
+    "For achivement the best result, audio should have number of channels and sample rate expected by model. If audio does not fulfit these requirements, it can be converted to desired sample rate and number of channels using `convert_audio` function. "
```

Comment:
To achieve the best result, audio should have the number of channels and sample rate expected by the model. If audio does not fulfill these requirements, it can be converted to the desired sample rate and the number of channels using `convert_audio` function.